src/vs/workbench/contrib/notebook/browser/diff/diffComponents.ts

```diff
@@ -142,9 +142,7 @@ class PropertyHeader extends Disposable {
 				return undefined;
 			}
 		}, this.menuService, this.contextKeyService, this.contextMenuService, this.keybindingService, this.commandService, this.telemetryService));
-		this._toolbar.context = {
-			cell: this.cell
-		};
+		this._toolbar.context = this.cell;
```

Comment:
@rebornix Making this change, as I'm not going to be passing any cells in Notebook Metadata menu contribution toolbars.